.clang-format

```diff
@@ -1,3 +1,5 @@
+# SPDX-FileCopyrightText: Helmholtz-Zentrum Dresden-Rossendorf e.V. <https://www.hzdr.de>
+# SPDX-License-Identifier: CC0-1.0
```

Comment:
`CC0-1.0` is essentially the public domain.

Reply:
Do you think it's necessary to also put these utility files under a license? I imagine their reusability severly limited.